pipelines/pipelines/maven/build-main.yaml

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright contributors to the Galasa project 
+# 
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: main-maven
+  namespace: galasa-pipelines
+spec:
+  workspaces:
+  - name: git-workspace
+  params:
+  - name: commit-sha
+    type: string
+  - name: merge-sha
+    type: string
+  tasks:
+  - name: clone-automation
+    taskRef: 
+      name: git-clone
+    params:
+    - name: url
+      value: https://github.com/galasa-dev/automation
+    - name: revision
+      value: main
+    - name: refspec
+      value: refs/heads/main:refs/heads/main
+    - name: depth
+      value: "99999999"
+    - name: subdirectory
+      value: $(context.pipelineRun.name)/automation
+    workspaces:
+     - name: output
+       workspace: git-workspace
+  - name: clone-maven
+    taskRef: 
+      name: git-clone
+    runAfter:
+      - clone-automation
+    params:
+    - name: url
+      value: https://github.com/galasa-dev/maven
+    - name: revision
+      value: main
+    - name: refspec
+      value: refs/heads/main:refs/heads/main
+    - name: depth
+      value: "99999999"
+    - name: subdirectory
+      value: $(context.pipelineRun.name)/maven
+    workspaces:
+     - name: output
+       workspace: git-workspace  
+  - name: get-commit
+    taskRef:
+      name: get-commit
+    runAfter:
+    - clone-wrapping
+    params:
+    - name: pipelineRunName
+      value: $(context.pipelineRun.name)
+    - name: repo
+      value: maven
+    workspaces:
+     - name: git-workspace
+       workspace: git-workspace
+  - name: maven-gpg
+    taskRef:
+      name: maven-gpg
+    runAfter:
+    - clone-wrapping 
+    params:
+    - name: context 
+      value: $(context.pipelineRun.name)/maven
+    workspaces:
+     - name: git-workspace
+       workspace: git-workspace
+  - name: main-maven-build-maven
+    taskRef:
+      name: maven-build
+    runAfter:
+    - maven-gpg
+    params:
+    - name: context
+      value: $(context.pipelineRun.name)/maven
+    - name: build-args
+      value: 
+        - "-Dgpg.skip=true"
+        - "-Dgalasa.source.repo=https://repo.maven.apache.org/maven2/"
+        - "-Dgalasa.central.repo=https://repo.maven.apache.org/maven2/"
+        - "-Dgalasa.release.repo=file:/workspace/git/$(context.pipelineRun.name)/maven/repo"
+    - name: command
+      value: deploy
+    workspaces:
+     - name: git-workspace
+       workspace: git-workspace
+  - name: main-docker-build-maven
+    taskRef:
+      name: docker-build
+    runAfter:
+    - get-commit
+    - main-maven-build-maven
```

Comment:
Same again - maven build task is not required in the main build so can remove please, and remove from runAfter in line 102